lib/tasks/link_checker.rake

```diff
@@ -18,16 +18,16 @@ end
 
 desc "link checker"
 task :link_checker, [:date] => [:environment] do |_, _args|
-  links = Dir.glob("config/locales/*.yml").map { |locale_filename| gather_urls YAML.safe_load(File.read(locale_filename)) }
-  links.flatten!.uniq!
-
-  gov_links = []
-  other_links = []
-  links.flatten.uniq.each do |uri|
-    if URI(uri).host.ends_with? ".gov.uk"
-      gov_links << uri
+  urls = Dir.glob("config/locales/*.yml").map { |locale_filename| gather_urls YAML.safe_load(File.read(locale_filename)) }
```

Comment:
Can this renaming be fixed up? It'll make the history easier to follow/access for the devs that come after us.